src/schemata/definitions/workflow_template/action_objects/getResources.yml

```
@@ -0,0 +1,46 @@
+---
+type: object
+title: Query Resources
+description: |
+  An action to make queries on the API/V2 resources 
+propertyNames:
+  enum:
+    - get_resources
+    - GetResources
+    - GET_RESOURCES
+additionalProperties:
+  type: object
+  required:
+    - tag
+  properties:
+    tag:
+      $ref: '#/definitions/resourceTag'
+    filter:
+      type: object
+      propertyNames:
+        $ref: '#/definitions/resourceTag'
+    page: 
+      type: object
+      propertyNames:
+        $ref: '#/definitions/resourceTag'
+    sort: 
+      type: object
+      propertyNames:
+        $ref: '#/definitions/resourceTag'
+    blocking:
+      type: boolean
+      description: |
+        Defines if the query action will block the subsequent steps
+      default: false
+
+example:
+  - getResources:
+      tag: workflow_runs
+      filter:
+        workflow_runs:
+          workflow_template_id: workflow_run.data.attributes.workflow_template_id
+      page:
+        workflow_runs:
+          size: 20
+      sort:
+        workflow_runs: "-created_at"
```

```suggestion
  - getResources:
      workflow_runs:
        filter:
          workflow_template_id: workflow_run.data.attributes.workflow_template_id
        page:
          size: 20
        sort: "-created_at"
```
Can we have something like this instead?

---

I am using it this way cause that is how the filter and pagination logic is currently structured.